src/SprykerEco/Zed/UnzerApi/Business/Api/Request/CreatePaymentResourceRequest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use Symfony\Component\HttpFoundation\Request;
+
+class CreatePaymentResourceRequest extends UnzerApiAbstractRequest implements UnzerApiRequestInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return string
+     */
+    public function getUrl(UnzerApiRequestTransfer $unzerApiRequestTransfer): string
+    {
+        return sprintf(
+            $this->unzerApiConfig->getUnzerApiCreatePaymentResource(),
+            $unzerApiRequestTransfer->getCreatePaymentResourceRequest()->getPaymentMethod()
```

Comment:
Please use `orFail` for methods inside cascade changing
e.g. `$unzerApiRequestTransfer-> getCreatePaymentResourceRequestOrFail()-> getPaymentMethod()`

Reply:
Fixed

Reply:
Please, add also `"spryker/transfer": "^3.25.0"` into composer.json.
And create file vendor/spryker/spryker/Bundles/ProductConfiguration/dependency.json with
```
{
    "include": {
        "spryker/transfer": "Provides transfer objects definition with `::get*OrFail()` functionality."
    }
}
```

Reply:
Fixed